.travis.yml

```diff
@@ -73,15 +73,9 @@ _base_envs:
   python: 3.9
   after_failure: skip
 python:
-- 3.5
```

Comment:
FWIW having Travis integration dead in this repo for months, there's no need to care about updating its config. Hopefully, I'll be able to reintroduce a proper CI setup per the best practices I've developed over the last year or so.